src/main/java/com/minecolonies/entity/ai/citizen/fisherman/Pond.java

```diff
@@ -120,15 +125,16 @@ private static boolean checkWaterPoolInDirectionZ(IBlockAccess world, int x, int
     private static boolean checkWaterPoolInDirectionZThenX(IBlockAccess world, int x, int y, int z, int vector)
     {
         //Check 6 blocks in direction +/- z
-        for (int dz = z + 6 * vector; dz <= z + 6 * vector; dz++)
+        for (int dz = z + WATER_POOL_WIDTH_REQUIREMENT * vector; dz <= z + WATER_POOL_WIDTH_REQUIREMENT * vector; dz++)
         {
             if (world.getBlockState(new BlockPos(x, y, dz)).getBlock() != Blocks.water)
             {
                 return false;
             }
         }
         //Takes the middle z block and searches 3 water blocks to both sides
-        return checkWaterPoolInDirectionX(world, x, y, z + 3 * vector, 1) && checkWaterPoolInDirectionX(world, x, y, z + 3 * vector, -1);
+        return checkWaterPoolInDirectionX(world, x, y, z + WATER_POOL_HEIGHT_REQUIREMENT * vector, 1) && checkWaterPoolInDirectionX(world, x, y, z + WATER_POOL_HEIGHT_REQUIREMENT
+                                                                                                                                                     * vector, -1);
     }
 
     /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)